ultra/docs/getting_started.md

```diff
@@ -80,7 +80,7 @@ After training your agent, your models should be saved under `logs/<timestamped_
   - `--level`: The level of the task (default is easy).
   - `--agents`: The ids of agents that you wish to evaluate (default is None; all agents are subject to evaluation).
   - `--episodes`: The number of evaluation episodes (default is 200).
-  - `--max-episode-steps`: The option to limit the number of steps per epsiodes (default is 200).
+  - `--max-episode-steps`: The option to limit the number of steps per epsiodes (default is 1200).
```

Comment:
I think this also has to be changed in line 54?

Reply:
Oh yeah, perfect thank you @christianjans